### ISSUE TITLE
Use Tox to simplify testing and linking

### ISSUE THREAD

**Issue by AKuederle** [learn-and-code/MoneyPy#1]
Set up a Tox config to automatically run tests against all supported Python configurations either locally or in CI

**Comment by AKuederle** [comment]
Any updates on this? :)

**Comment by fabianhough** [comment]
Yes!  Running some tests on my own branch, to make sure it works.

**Comment by AKuederle** [comment]
Cool! If you want just start a pull request with your branch, and I will have a look :)
